src/validation-form/js/scripts.js

```diff
@@ -267,7 +267,7 @@ function apiError(data, textStatus, jqXHR) {
 function submitValidationRequest() {
 	if(verifyForm()) {
 		$.ajax({
-			url: '/validate/',
+			url: 'http://localhost:5000/validate/',
```

Comment:
This should not be changed, it makes the application less portable.

Reply:
Fixed.  Ok to merge?